controllers/cron_controller.py

```diff
@@ -666,6 +689,29 @@ def get(self):
             logging.exception(e)
 
 
+class CheckMediaDo(webapp.RequestHandler):
+    """
+    Remove unavailable media
+    """
+    def get(self, media_key):
```

Comment:
Everything between `.get()` and `.put()` should be in a transaction (https://cloud.google.com/appengine/docs/standard/python/ndb/transactions). Right now, this runs the risk of losing data (although the risk is really small). If between `.get()` and `.put()`, another process edits the Media object (like an admin adding a tag), `.put()` will overwrite the update.